service-base/src/main/java/org/eclipse/hono/service/http/HttpServiceBase.java

```diff
@@ -52,6 +52,8 @@
      */
     private AuthenticationHandler authHandler;
 
+    private final HttpServiceConfigProperties httpServiceConfigProperties = new HttpServiceConfigProperties();
```

Comment:
I think the changes in this class shouldn't be there - see https://github.com/eclipse/hono/pull/3277#discussion_r886765745.